app/organisation/accept_organisation_invite.py

```diff
@@ -0,0 +1,30 @@
+from flask import Blueprint, jsonify, current_app
+from itsdangerous import SignatureExpired
+from notifications_utils.url_safe_token import check_token
+
+from app.dao.organisation_dao import dao_get_invited_organisation_user
+from app.errors import register_errors, InvalidRequest
+
+accept_organisation_invite_blueprint = Blueprint(
+    'accept_organisation_invite', __name__,
+    url_prefix='/organisation-invitation')
+
+register_errors(accept_organisation_invite_blueprint)
+
+
+@accept_organisation_invite_blueprint.route("/<token>", methods=['GET'])
```

Comment:
We can combine the two accept invite blueprint with the "any" parameter
@accept_invite_blueprint.route("/<any:accept_invite, accept_organisation_invite: level>/<token>", methods=['GET'])
def accept_invitation(level, token):

then "level" will be available to be used in the function.

Reply:
Similar applies to the invite rests.

Reply:
You will end up with maintaining two flows, which pretty much does the same thing.